tests/suites/dashboard/dashboard.sh

```
@@ -9,12 +9,12 @@ run_dashboard_deploy() {
 	wait_for "controller" "$(active_condition "dashboard")"
 	juju wait-for application dashboard
 	sleep 10 # short wait for relation data to update
-	open_dashboard
+	# open_dashboard
```

We could test this throws an error, rather than ignoring the command.